pkg/conf/config.go

```diff
@@ -35,3 +41,35 @@ func GetBuilderKey() (string, error) {
 	builderKey := string(builderKeyBytes)
 	return builderKey, nil
 }
+
+func GetStorageParams(env sys.Env) (Parameters, error) {
```

Comment:
Ok, I see what you mean. I think the argument could be made either way, so I won't contest your decision.